src/GraphNodeManagerViewExtension/ViewModels/GridNodeViewModel.cs

```diff
@@ -364,21 +364,13 @@ public Guid NodeGuid
         /// <returns></returns>
         private bool IsNodeEmptyList(MirrorData mirrorData)
         {
-            if (mirrorData == null || !mirrorData.IsCollection) return false;
+            if (mirrorData.IsNull || mirrorData == null || !mirrorData.IsCollection) return false;
 
-            try
```

Comment:
I don't think there is anything wrong with keeping these try catches in areas of code where you know it is hard to avoid null refs.

Do you expect many exceptions? If the exceptions are not constant, adding a try is not a large performance cost.
https://stackoverflow.com/questions/1308432/do-try-catch-blocks-hurt-performance-when-exceptions-are-not-thrown

Reply:
The information regarding the performance cost of try catches is awesome! I will use that in the future, but I think, for this particular case, it was the unhandled exception that was causing the issue and made me use try/catch blocks before your comments. Fixing the root cause solves the problem altogether, I think!